osbuild/meta.py

```diff
@@ -409,6 +409,42 @@ def _parse_caps(cls, _klass, _name, node):
 
     @classmethod
     def load(cls, root, klass, name) -> Optional["ModuleInfo"]:
+        base = cls.MODULES.get(klass)
+        if not base:
+            raise ValueError(f"Unsupported type: {klass}")
+        path = os.path.join(root, base, name)
+
+        try:
+            return cls._load_from_json(path, klass, name)
+        except FileNotFoundError:
+            # should we print a deprecation warning here?
+            pass
+        return cls._load_from_py(path, klass, name)
```

Comment:
I'm wondering if it would make sense to warn / fail, in case the module has the schema defined in both, the external JSON file and in its Python implementation.